R/blpConnect.R

```diff
@@ -45,18 +47,23 @@
 ##' environment. This effectively frees users from having to
 ##' explicitly create such an object.
 ##' @author Whit Armstrong and Dirk Eddelbuettel
-##' @seealso Many SAPI and bPipe connections require authentication 
+##' @seealso Many SAPI and bPipe connections require authentication
 ##' via \code{blpAuthenticate} after \code{blpConnect}.
 ##' @examples
 ##' \dontrun{
 ##'   con <- blpConnect()   # adjust as needed
 ##' }
 blpConnect <- function(host=getOption("blpHost", "localhost"),
                        port=getOption("blpPort", 8194L),
-                       default=TRUE) {
+                       default=TRUE,
+                       app_name = getOption("blpAppName", NULL)) {
```

Comment:
Stylistic nitpick: we should use something other than snake_case for argument names. We've tended toward camelCase in most functions, thought there are also a few old-school R dot.separated names left over. I think `appName` fits best here.